AntistasiOfficial.Altis/distancias3.sqf

```diff
@@ -95,6 +95,6 @@ while {true} do {
 			};
 		};
 
-	} forEach (markers + countSupplyCrates);
+	} forEach (markers + markerSupplyCrates);
```

Comment:
It tells me markerSupplyCrates is undefined variable

Reply:
it means that this function is executed before the definition of the global markerSupplyCrates.
just wrap this loop in a condition with a statement like `is (isNil markerSupplyCrates)` or maybe isNull ?

Reply:
yes fixed